pladder/bjbot.py

```diff
@@ -0,0 +1,31 @@
+import os
+from pladder.plugin import Plugin
+
+class BjBotPlugin(Plugin):
+    def __init__(self, bot):
+        super().__init__()
+        self._datorbas = dict()
+        self.importera(self._datorbas, os.path.join(os.path.dirname(__file__), "triggers",
```

Comment:
Also, can you change the code to work even when there isn't a trigger file? Otherwise other people will have troubles running the bot.